### ISSUE TITLE
4.0.0 breaks requiring ts-node/register on Windows by Gulp

### ISSUE THREAD

**Issue by hach-que** [TypeStrong/ts-node#479]
According to a lot of documentation out there, you only need to install `ts-node` alongside Gulp, and Gulp will be able to execute `Gulpfile.ts` directly via `ts-node/register`. The 4.0.0 package doesn't seem to include this file, and this breaks that integration with:

```
[10:02:19] Failed to load external module ts-node/register
[10:02:19] Failed to load external module typescript-node/register
[10:02:19] Failed to load external module typescript-register
[10:02:19] Failed to load external module typescript-require
```

I was super confused when I couldn't find any information on this issue, only to realise the 4.0.0 is only 30 minutes old and it's likely I'm just the first to hit this issue with the new major release.

Rolling back to `3.3.0` in `package.json` allows Gulp with TypeScript to work correctly.

**Comment by blakeembrey** [comment]
Thanks for the catch! Forgot to update the `package.json` files list.

**Comment by blakeembrey** [comment]
Fixed with 4.0.1 😄
